Resources/doc/7-messenger-integration.md

```diff
@@ -0,0 +1,58 @@
+# Messenger integration
+
+If you have installed [Symfony Messenger](https://symfony.com/doc/current/messenger.html#installation), then you can 
+dispatch `Presta\SitemapBundle\Messenger\DumpSitemapMessage` message to your transport to handle it asynchronously or
+synchronously.
+
+## Routing the message to your transport
+
+```yaml
+# config/packages/messenger.yaml
+framework:
+    messenger:
+        transports:
+            async: "%env(MESSENGER_TRANSPORT_DSN)%"
+
+        routing:
+            # async is whatever name you gave your transport above
+            'Presta\SitemapBundle\Messenger\DumpSitemapMessage':  async
+```
+
+After configuring the message routing dispatch the message like this:
+
+```php
+// src/Controller/DefaultController.php
+namespace App\Controller;
+
+use Presta\SitemapBundle\Messenger\DumpSitemapMessage;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+class DefaultController extends AbstractController
+{
+    public function index(MessageBusInterface $bus)
```

Comment:
Since the rework of our tests, we introduced an integration test suite (which is a minimal symfony project).
I'd like this snippet to be integrated in a test of that suite, could you write some test like that ?

Reply:
I've added some integration tests :)